ads/revcontent.md

```diff
@@ -0,0 +1,41 @@
+<!---
+Copyright 2015 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Revcontent
+
+## Example
+
+```html
+<amp-ad width=300 height=1000
+        type="revcontent"
+        data-wrapper="rcjsload_2ff711"
+        data-endpoint="trends.revcontent.com"
+        data-ssl="true"
+        data-id="203">
+
+</amp-ad>
+```
+
+## Configuration
+
+For semantics of configuration, please see ad network documentation.
+
+Supported parameters:
+
+- wrapper. Must be a valid dom.div id string
+- endpoint. Must be a valid Trends Server *required*
+- id. Must be a valid Widget ID *required* 
+- ssl. Must be set to "true" for SSL mode *required*
```

Comment:
Would it be ok if I remove **`ssl`** as user-facing option from our readme? **SSL** will still default to true if not provided (Pending a deploy on our CDN for our service file), so this parameter would be used to turn SSL off in most cases.